tools/e2etesting/steps/variables.yml

```diff
@@ -5,4 +5,5 @@ variables:
   AgentPool: Azure-IoT-Manufacturing
   IAILocalFilename: Microsoft.Azure.IIoT.Deployment.exe
   TestPath: $(System.DefaultWorkingDirectory)\e2e-tests\IIoTPlatform-E2E-Tests
-  Runtime: 'win-x64'
\ No newline at end of file
+  Runtime: 'win-x64'
+  KeysPath: $(System.DefaultWorkingDirectory)\keys
```

Comment:
See above.

Reply:
changed to use $(Agent.TempDirectory) , which is by definition cleaned after each build